### ISSUE TITLE
Mixed JSON files

### ISSUE THREAD

**Issue by i2amroy** [CleverRaven/Cataclysm-DDA#6481]
I'm all for being able to maintain and handle multiple types of JSON items regardless of their type in a single file, and for splitting some of our larger JSON files up, but something I'm disliking is how we're quickly degenerating into forcing someone to search through every single JSON file to track down anything they want to edit.

For example the boxing PR (which looks awesome BTW) adds 2 new types of furniture into the new "boxing.json" file. Prior to that being merged if I wanted to add a new furniture flag I'd have to check the furniture.json file and that would be it. After its merging I would need to not only check the furniture.json file, I'd also need to run a check through all other json files for furniture JSON entries.

In short, we don't need perfect file type organization for the JSON files, but I would like some form of organization present, if only as a "best practice" type of thing and to allow for future developers to be more efficient in their additions. Ideas?

**Comment by illi-kun** [comment]
This was fixed by Rivet in #10642.
